R/db-odbc-oracle.R

```diff
@@ -0,0 +1,235 @@
+#' @export
+sql_select.Oracle<- function(con, select, from, where = NULL,
```

Comment:
I like that idea.  It looks like most of the clauses follow the same pattern, should I target a `sql_select_clause()` function that takes the clause as an argument?  So it may work like `out$group_by <- sql_select_clause("GROUP BY", group_by)`?